src/TorchSharp/Tensor/Tensor.Math.cs

```diff
@@ -548,6 +548,81 @@ public Tensor ceil_()
                 return new Tensor(res);
             }
 
+            [DllImport("LibTorchSharp")]
+            static extern IntPtr THSTensor_conj(IntPtr tensor);
+
+            /// <summary>
+            /// Returns a view of input with a flipped conjugate bit. If input has a non-complex dtype, this function just returns input.
+            /// </summary>
+            /// <returns></returns>
+            public Tensor conj()
+            {
+                var res = THSTensor_conj(Handle);
+                if (res == IntPtr.Zero)
+                    torch.CheckForErrors();
+                return new Tensor(res);
```

Comment:
Not for this PR, but would not it be nice to have a function, that combines result check with `Tensor` creation?

Then most of these `Tensor` functions would be one-liners. E.g. `=> Tensor.FromHandleOrError(THSTensor_conj(Handle))`